backend/api/v2beta1/run.proto

```diff
@@ -76,13 +84,27 @@ service RunService {
     };
```

Comment:
Suggest 
```get: "/apis/v2beta1/runs/{run_id}"```
 to become: 
``` get: "/apis/v2beta1/runs/{id}"```

Thus be consistent with **achive**, **unarchive** endpoints.

Reply:
There are other endpoints that use `run_id` instead of just `id` - like **ReadArtifact**, **TerminateRun**, **ReportRunMetrics**. I suggest using just `id`. Only for **ReadArtifact** I do not have a strong opinion. Probably only for _ReadArtifacts_ it makes sense to use `run_id`, and prevent confusion since there are other parameters.

Reply:
Make sense; updated all endpoints to use just `id` except ReadArtifact for the same reason that you shared above.